docs/data-model.md

```
@@ -0,0 +1,349 @@
+  description: string
+  category: string
+  xpReward: number
+  icon?: string
+  requirements: JSON // flexible criteria
+  isHidden: boolean
+}
+```
+
+## Key Relationship Patterns
+
+### Hierarchical Relationships
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)